src/components/close-button/__tests__/CloseButton.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
I believe our policy is to compose new component-level unit tests using @testing-library/react (RTL) instead of enzyme. (That is because enzyme usually targets implementation details, whereas RTL forces a user-perspective approach to tests.) I've gotten pretty decent with RTL over the last few weeks, so if you want any guidance on this, I would be happy to help out :) 

For reference:
My most recent RTL tests are in my latest PR, e.g.: https://git.dev.box.net/Box/EndUserApp/pull/8121/files#diff-0de84049a19f96b8eca357bf3a4b566ae22a135dd1711513db91f83425dc1c87

In EUA, you could check out /src/components/core/left-nav/components/__tests__/LeftNav.test.js
In BUIE, there is also my first RTL work: src/features/collapsible-sidebar/__tests__/CollapsibleSidebar.test.js